app/liquid_tags/asciinema_tag.rb

```diff
@@ -0,0 +1,32 @@
+class AsciinemaTag < LiquidTagBase
+  PARTIAL = "liquids/asciinema".freeze
+
+  def initialize(tag_name, id, tokens)
+    super
+    @id = parse_id(id)
+  end
+
+  def render(_context)
+    ActionController::Base.new.render_to_string(
+      partial: PARTIAL,
+      locals: {
+        id: @id
+      },
+    )
+  end
+
+  private
+
+  def parse_id(input)
+    input_no_space = input.delete(" ")
+    raise StandardError, "Invalid Asciinema Id" unless valid_id?(input_no_space)
```

Comment:
I'd prefer an error message that showed which ID is erroring out:
```suggestion
    raise StandardError, "Invalid Asciinema ID: {% asciinema #{input_no_space} %}" unless valid_id?(input_no_space)
```

Reply:
None of the other Liquid tags in `app/liquid_tags` seem to include the ID in the corresponding error, so I opted for consistency. I have no strong feelings though, so if you want it included that's fine.

Reply:
Yeah, that's more our fault of not improving all the error messages. I think a general trend of making error messages more informative is a good direction.

Reply:
👍 done